internal/landing/html/index.html

```diff
@@ -34,12 +34,6 @@ <h2>Details</h2>
             Each peer receives information back about other affiliates from the discovery service, decrypts it and uses it to drive KubeSpan and cluster discovery.
         </p>
 
-        <p>
-            Moreover, the discovery service has no peristence.
```

Comment:
I think we dropped too much here. The first sentence doesn't apply, but the second might be rephrased with `Data is stored in memory with TTL, and periodic snapshots to disk are enabled.`